src/main/java/org/datadog/jmxfetch/App.java

```diff
@@ -945,6 +955,34 @@ public TaskStatusHandler invoke(
         }
     }
 
+    private Map<String,Object> getTelemetryInitConfig() {
+        Map<String,Object> config = new HashMap<String,Object>();
+        config.put("is_jmx",true);
+        return config;
+    }
+
+    private Map<String,Object> getTelemetryInstanceConfig() {
+        Map<String,Object> config = new HashMap<String,Object>();
+        config.put("name","jmxfetch_telemetry_instance");
+        config.put("collect_default_jvm_metrics",true);
+        config.put("new_gc_metrics",true);
+        config.put("process_name_regex",".*org.datadog.jmxfetch.App.*");
+
+        List<Object> conf = new ArrayList<Object>();
+        Map<String,Object> confMap = new HashMap<String,Object>();
+        Map<String,Object> includeMap = new HashMap<String,Object>();
+        includeMap.put("domain","jmxfetch");
+        confMap.put("include", includeMap);
+        conf.add(confMap);
+        config.put("conf",conf);
+
+        List<String> tags = new ArrayList<String>();
+        tags.add("jmx:fetch");
```

Comment:
For this PR, lets not put any tags, just keep it as an empty list. This tag isn't particularly useful, I think this originated from some testing code.

In a follow-up PR, I'd like you to work on adding a tag that reflects the current version of JMXFetch. So `tags.add("jmxfetch_version:0.47.10")`.